graphqlapi/src/main/java/io/stargate/graphql/schema/fetchers/dml/DmlFetcher.java

```diff
@@ -101,6 +101,21 @@ protected DmlFetcher(
         Relation relation;
         if (condition.getKey().equals("in")) {
           relation = relationStart.in(buildListLiterals(column, condition.getValue()));
+        } else if (condition.getKey().equals("contains")) {
+          relation = relationStart.contains(toCqlElementTerm(column, condition.getValue()));
+        } else if (condition.getKey().equals("containsKey")) {
+          relation = relationStart.containsKey(toCqlKeyTerm(column, condition.getValue()));
+        } else if (condition.getKey().equals("containsValue")) {
+          relation = relationStart.contains(toCqlValueTerm(column, condition.getValue()));
+        } else if (condition.getKey().equals("containsEntry")) {
+          Column.ColumnType mapType = column.type();
+          assert mapType != null && mapType.isMap();
+          Map<String, Object> entry = (Map<String, Object>) condition.getValue();
+          Column.ColumnType keyType = mapType.parameters().get(0);
+          Term keyTerm = toCqlTerm(keyType, entry.get("key"));
+          Column.ColumnType valueType = mapType.parameters().get(1);
+          Term valueTerm = toCqlTerm(valueType, entry.get("value"));
+          relation = Relation.mapValue(column.name(), keyTerm).isEqualTo(valueTerm);
```

Comment:
There's a `FilterOperator` enum screaming to be extracted here, and reused in `FieldFilterInputTypeCache`. But this PR is already big, I'll probably keep it for a follow-up ticket.